core/src/main/scala/kafka/security/auth/ResourceType.scala

```diff
@@ -31,30 +32,35 @@ case object Topic extends ResourceType {
   val name = "Topic"
   val error = Errors.TOPIC_AUTHORIZATION_FAILED
   val toJava = JResourceType.TOPIC
+  val supportedOperations = Set(Read, Write, Create, Describe, Delete, Alter, DescribeConfigs, AlterConfigs, All)
```

Comment:
Looking at the places where this is used, it looks like it would be better not in include `All` in `supportedOperations`. So this would just be actual supported operations and `AclCommand` can add `All` to the set to create its `validOps` set. What do you think?

Reply:
makes sense. Update the PR.